actor/builtin/miner/miner_test.go

```diff
@@ -418,6 +421,21 @@ func TestMinerSubmitPoStSectorSetUpdates(t *testing.T) {
 	require.NoError(t, res.ExecutionError)
 	require.Equal(t, uint8(0), res.Receipt.ExitCode)
 
+	t.Run("proving-set contains only the first sector committed before submitPoSt", func(t *testing.T) {
```

Comment:
Is it better for me to break off the same `commitSector` and `submitPoSt` flow into another test that only tests power?

Reply:
Probably yes. Adding more to this chain of update and assertions will get hard to maintain